web_widget_slickroom/static/description/icon.svg

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file should not be included unless the icon you are adding is an original work. It's meant for if someone needs to edit the icon, but we store the svg in the template so it's not needed here & just adds overhead to the diff